lms/djangoapps/lti_provider/models.py

```diff
@@ -0,0 +1,14 @@
+"""
+Database models for the LTI provider feature.
+"""
+from django.db import models
+
+
+class LtiConsumer(models.Model):
```

Comment:
Huh. I had forgotten that line in our coding standards and was basing this off of my recollection of PEP8 (_"When using abbreviations in CapWords, capitalize all the letters of the abbreviation. Thus HTTPServerError is better than HttpServerError."_). A quick scan through some class names I remember in platform shows that some older stuff is in the PEP8 style, but the newer stuff is in the style you have. So yeah, let's keep your class names as is, and sorry for the misdirection. I should have figured that @sarina would have flagged you on it if it needed changing.